mkosi: add page (#10721)
tldr-pages/tldr: pages/linux/mkosi.md

Reviewer: line 10: `distribuion`

Author: good catch, thanks.

Reviewer: ok!

# mkosi

> > Tool for building modern, legacy-free Linux images. Part of systemd.

Reviewer 2: ```suggestion
> Tool for building modern, legacy-free Linux images.
> Part of `systemd`.
```

> > More information: <https://github.com/systemd/mkosi>.

Reviewer 3: manned.org's page is more useful.
```suggestion
> More information: <https://manned.org/mkosi>.
```

- Show current build configuration to verify what would be built:

`mkosi summary`

- Build an image with default settings, if no distribution is selected, distribution of the host system is used:

`mkosi build --distribution={{fedora|debian|ubuntu|arch|opensuse}}`

- Build image and get interactive shell in a systemd-nspawn container of the image:

`mkosi shell`

> - Boot image in a virtual machine using qemu, only supported for disk images or cpio images when a kernel is provided:

Reviewer 2: ```suggestion
- Boot image in a virtual machine using QEMU, only supported for disk images or CPIO images when a kernel is provided:
```

`mkosi qemu`

> - Get more help:

Reviewer 2: ```suggestion
- Display help:
```
We use this generic text for help descriptions across our pages.

# mkosi

> Tool for building modern, legacy-free Linux images.
> Part of `systemd`.
> More information: <https://manned.org/mkosi>.

- Show current build configuration to verify what would be built:

`mkosi summary`

> - Build an image with default settings, if no distribution is selected, distribution of the host system is used:

Reviewer 4: ```suggestion
- Build an image with default settings (if no distribution is selected, the distribution of the host system is used):
```

> `mkosi build --distribution={{fedora|debian|ubuntu|arch|opensuse}}`

Reviewer 4: ```suggestion
`mkosi build --distribution {{fedora|debian|ubuntu|arch|opensuse|...}}`
```

> - Build image and get interactive shell in a systemd-nspawn container of the image:

Reviewer 4: ```suggestion
- Build an image and run an interactive shell in a systemd-nspawn container of the image:
```

`mkosi shell`

> - Boot image in a virtual machine using QEMU, only supported for disk images or CPIO images when a kernel is provided:

Reviewer 4: ```suggestion
- Boot an image in a virtual machine using QEMU (only supported for disk images or CPIO images when a kernel is provided):
```
